### PR TITLE
[HOPSWORKS-1817] Replace influxdb with prometheus + pushgateway for Spark applications

### PR DESCRIPTION
**Pull request by SirOibaf**

### CODE REVIEW

File: templates/default/pushgateway.service.erb [hunk 1]
```diff
@@ -0,0 +1,21 @@
+[Unit]
+Description = Prometheus
```
Comment (tkakantousis) [inline templates/default/pushgateway.service.erb:2]: Copy-paste leftover?

File: templates/default/pushgateway.service.erb [hunk 2]
```diff
@@ -0,0 +1,21 @@
+[Unit]
+Description = Prometheus  
+After = syslog.target network.target remote-fs.target
```
Comment (tkakantousis) [inline templates/default/pushgateway.service.erb:3]: Is there any dependency of prometheus on pushgateway?

**Review by tkakantousis — APPROVED** [review]

File: templates/default/pushgateway.service.erb (hunk 2 above)
Comment (SirOibaf) [reply templates/default/pushgateway.service.erb:3]: No, the entire architecture is made to be independent. Prometheus keeps scraping the targets, no matter if they are available or not.

File: templates/default/pushgateway.service.erb (hunk 1 above)
Comment (SirOibaf) [reply templates/default/pushgateway.service.erb]: done

**Pull request merged**